src/pages/register.tsx

```diff
@@ -0,0 +1,146 @@
+import { DatePicker } from "@mui/x-date-pickers/DatePicker";
+import CommonTextField from "@/components/public/CommonTextField";
+import CommonDropdown from "@/components/public/CommonDropdown";
+
+export default function Home() {
+  const [email, setEmail] = React.useState("");
+  const [password, setPassword] = React.useState("");
+  const [confirmPassword, setConfirmPassword] = React.useState("");
+  // const [isSubmit, setIsSubmit] = React.useState(false);
+  const [value, setValue] = React.useState<Dayjs | null>(null);
+  const [displayName, setDisplayName] = React.useState<string>("");
+  const [gender, setGender] = React.useState<string>("");
+
+  const handleDisplayNameChange = (
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void => {
+    setDisplayName(event.target.value);
+  };
+
+  function handleEmailChange(
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void {
+    setEmail(event.target.value);
+  }
+
+  function handlePasswordChange(
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void {
+    setPassword(event.target.value);
+  }
+
+  function handleConfirmPasswordChange(
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ): void {
+    setConfirmPassword(event.target.value);
+  }
+
+  function handleGenderChange(event: SelectChangeEvent<string>): void {
+    setGender(event.target.value);
+  }
+
+  const handleSelectChange = (event: SelectChangeEvent): void => {
+    setGender(event.target.value as string);
+  };
+
+  return (
+    <Stack spacing={3} alignItems="center" justifyContent="center" style={{ minHeight: "100vh" }}>
+      <Logo width={119} height={119} />
+
+      <Box style={{ width: "50vw" }}>
+        <CommonTextField
+          header="Username"
+          placeholder="Display Name"
+          value={displayName}
+          handleValueChange={handleDisplayNameChange}
+          char_limit={100}
+          isErr={false}
+          errMsg=""
+        />
+      </Box>
+
+      <Box style={{ width: "50vw", marginTop: 0 }}>
+        <CommonTextField
+          header="Email"
+          placeholder="Email"
+          value={email}
+          handleValueChange={handleEmailChange}
+          isErr={false}
+          errMsg=""
+        />
+      </Box>
+
+      <Box style={{ width: "50vw" }}>
+        <PasswordTextFeild
+          header="Password"
+          placeholder="Password"
+          value={password}
+          handleValueChange={handlePasswordChange}
+          isErr={false}
+          errMsg=""
+        />
+      </Box>
+
+      <Box style={{ width: "50vw" }}>
+        <PasswordTextFeild
+          header="Confirm Password"
+          placeholder="Confirm Password"
+          value={confirmPassword}
+          handleValueChange={handleConfirmPasswordChange}
+          isErr={false}
+          errMsg=""
+        />
+      </Box>
+
+      <Grid item>
+        <Grid container spacing={2}>
```

Comment:
The items in the container is (for now) not the same size and as in figma. Please fix it.

Reply:
still not the same as figma. (I send the vdo to you on discord)